mscore/palette.cpp

```diff
@@ -1163,6 +1163,45 @@ bool Palette::event(QEvent* ev)
       }
 
 //---------------------------------------------------------
+//   keyPressEvent
+//---------------------------------------------------------
+
+void PaletteScrollArea::keyPressEvent(QKeyEvent *event)
+      {
+      NewWizard* nw = mscore->getNewWizard();
```

Comment:
mmm... this is ugly :-/ 
It would be good if palette wouldn't need to know anything about the wizard or its page...